recipes/rna-seqc/meta.yaml

```diff
@@ -1,27 +1,28 @@
-{% set version = "1.1.8" %}
+{% set version = "2.3.3" %}
+{% set base = "https://github.com/broadinstitute/rnaseqc/releases/download" %}
+{% set sha256linux = "bfe5d28b900713a7286ebcd5323db56bbec95c3848e8d74f4e1cfdb883840669" %}
+{% set sha256osx = "8fe748a5f345c8b71519161edbff1ad1a995a68edbfbe7a0994ea01c857bf7c3" %}
 
 package:
   name: rna-seqc
-  version: {{ version }}
+  version: '{{ version }}'
 
 source:
-  url: http://www.broadinstitute.org/cancer/cga/tools/rnaseqc/RNA-SeQC_v1.1.8.jar
-  md5: 71d7b5d3b3dcc1893cdc7f6819185d41
+  url: '{{base}}/v{{version}}/rnaseqc.v{{version}}.linux.gz'   # [linux]
+  sha256: '{{sha256linux}}'                                    # [linux]
 
-build:
-  number: 2
-  noarch: generic
+  url: '{{base}}/v{{version}}/rnaseqc.v{{version}}.osx.gz'     # [osx]
+  sha256: '{{sha256osx}}'                                      # [osx]
 
-requirements:
```

Comment:
You need to define the requirements. If this is now a python only package you should have python as run-requirement and pip and python as host requirement.

Reply:
Thanks very much for the review @bgruening. These are just static executables/binaries so there are no host or run requirements. I am of course happy to add an empty requirements block if needed, but I think that is redundant.

Reply:
Is it possible to build it from source? We recommend building from source to stay compatible with various old architectures.

Reply:
Yes - there's a "full source" tarball that includes the SeqLib submodule/dependency and some test data, IIRC. I'm happy to give this a go tomorrow. Cheers.

Reply:
Sorry - I think I see the problem with the test. Will try and fix tonight/tomorrow and let you know when I'm done.

Reply:
Done; but a code review for this guy would be very much appreciated. There are also a couple of python scripts that come bundled alongside the source and these have quite a few dependencies. Should these also be included in the build?

Reply:
The question is mostly how vital those are. A lot of tools come with random R/python helper scripts that are never used. We can focus on getting the primary package working and fix helper scripts as the need arises.